### PR TITLE
Add Get Help and Community Chat Pages

### PR DESCRIPTION
**Pull request by BenLloydPearson**
# Summary
This PR adds two pages that have important information I'd like to reference in other docs pages.

* Add a Get Help page that lists all of the resources to find help with Mattermost
* Add a community chat page that provides info about our community chat server and lists popular channels.
* Add both pages to the left navigation. This replaces the link to the support knowledge base because that info is shared in the new Get Help page.

# Ticket Link
This replaces #4806 and includes updates based on the feedback provided in that PR.

### CODE REVIEW

**Comment by BenLloydPearson** [comment]
@cwarnermm @justinegeffen I think this is how you intended for me to submit this PR originally. Please let me know if I need to make further adjustments.

**Comment by justinegeffen** [comment]
This is perfect, thank you so much @BenLloydPearson!! :)

**Comment by justinegeffen** [comment]
Off the top of my head, looking at: http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/4870/, I wonder if we can remove the chat tile and rather have it linked in the site banner/header? I don't know if we can balance out 9 tiles and this may be more visually obvious to readers.

The content would remain in the LHS nav in the site (pretty sure we can have it there without needing a landing page tile, @cwarnermm?)

File: source/guides/community-chat.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Mattermost Community Chat Server
+================================
+
+Mattermost hosts a chat server at `community.mattermost.com <https://community.mattermost.com/login>`__ to connect our team with our larger contributor and user community. This server is open to everyone, and we welcome anyone who wants to participate in our community to sign up.
```
Comment (justinegeffen) [inline source/guides/community-chat.rst:4]: Is "chat" the right wording in this case? Maybe "collaboration server" could work?

File: source/guides/community-chat.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+We have many channels on our community chat server you can join. Here are some of the more popular channels.
+
+**General Purpose Channels**
```
Comment (justinegeffen) [inline source/guides/community-chat.rst:8]:
```suggestion
**General purpose channels**
```

File: source/guides/community-chat.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- `Peer-to-peer Help: <https://community.mattermost.com/core/channels/peer-to-peer-help>`__ Get help from other community members on using, deploying, or contributing to Mattermost.
+- `Ask Anything: <https://community.mattermost.com/core/channels/ask-anything>`__ Ask general questions about Mattermost.
+
+**Contributor Discussions**
```
Comment (justinegeffen) [inline source/guides/community-chat.rst:13]:
```suggestion
**Contributor discussions**
```

File: source/guides/community-chat.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- `Developers: <https://community.mattermost.com/core/channels/developers>`__ Get help with development issues from our team and other developers in our community.
+- `Developers Meeting: <https://community.mattermost.com/core/channels/developers-meeting>`__ Join the weekly developers meeting to find out what Mattermost developers are working on.
+- `Contributors: <https://community.mattermost.com/core/channels/tickets>`__ Connect with other people who are working to make Mattermost better.
+- `Contributor Guilds: <https://developers.mattermost.com/contribute/getting-started/guilds/>`__ Find focused groups within our community that work on specific portions of the Mattermost ecosystem.
+- `Documentation Working Group: <https://community.mattermost.com/core/channels/dwg-documentation-working-group>`__ Provide feedback about Mattermost documentation and ask questions to the Mattermost Technical Writing team.
+
+**Social Channels**
```
Comment (justinegeffen) [inline source/guides/community-chat.rst:21]:
```suggestion
**Social channels**
```

**Review by justinegeffen — COMMENTED** [review]
Minor corrections to heading casing but otherwise this looks good editorially - I raised a separate point about these guides' place in the LHS and on the landing page.

**Comment by BenLloydPearson** [comment]
> Off the top of my head, looking at: http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/4870/, I wonder if we can remove the chat tile and rather have it linked in the site banner/header? I don't know if we can balance out 9 tiles and this may be more visually obvious to readers.

I'm fine with removing the tile from the home page, but the header isn't a good fit because there is a separate effort that's ongoing to standardize the headers across all of our sites.

File: source/guides/community-chat.rst (hunk 1 above)
Comment (BenLloydPearson) [reply source/guides/community-chat.rst]: I updated the language in this.

**Review by cwarnermm — APPROVED** [review]
Thank you so much! This is a great addition to the docs!

**Comment by cwarnermm** [comment]
Great point about the growing number of tiles on the homepage, @justinegeffen. If we're able to keep it in the LHS without having the tile on the homepage, that's ideal. My recommendation is to merge this as-is now. I suspect that the header and homepage are about to undergo a fair number of changes, and it might be valuable to revisit flow and navigation when we're a bit further along with Ben's updates. Thoughts?

**Comment by BenLloydPearson** [comment]
@cwarnermm is correct. We're in the process of building an entirely new landing page for the documentation home, so anything we change on that page now will likely be changed again in September.

**Comment by justinegeffen** [comment]
I'm on-board with what you and Ben have mentioned, I didn't think of it from that angle. Let's merge this and iterate. :)

**Pull request merged**